Chore/release workflow, README updates (#23)
valory-xyz/IEKit: README.md

An toolkit for autonomous services implementing a decentralized Impact Evaluator built with the [Open Autonomy](https://docs.autonolas.network/open-autonomy/) framework.

The demo service for the IEKit tracks mentions of [@autonolas](https://twitter.com/autonolas) on Twitter, assigns scores to them and writes those scores to a [Ceramic](https://ceramic.network/) stream.

> 

Author: I just left the requirements at the bottom to replicate the status of contribution service. For building the service the instructions already live on the docs.

An toolkit for autonomous services implementing a decentralized Impact Evaluator built with the [Open Autonomy](https://docs.autonolas.network/open-autonomy/) framework.

The demo service for the IEKit tracks mentions of [@autonolas](https://twitter.com/autonolas) on Twitter, assigns scores to them and writes those scores to a [Ceramic](https://ceramic.network/) stream.

> ## Running a demo the service

Reviewer: ```suggestion
## Running a demo service
```

Author: There was a spelling issue so I included this change myself directly

> ## Running the service

Reviewer: ```suggestion
## Running a demo the service
```

Reviewer: Place this section above, to be consistent with the sectioning order in https://docs.autonolas.network/product/iekit/

Author: Fixed

> To learn how to run the service, check the [IEKit tutorial](https://docs.autonolas.network/product/iekit/).

Reviewer: ```suggestion
To learn how to run a demo service based on the IEKit, read the [IEKit techncical docs](https://docs.autonolas.network/product/iekit/).
```
